### PR TITLE
Xskipper proposal

### PR DESCRIPTION
**Pull request by oshritf**
Creating proposal directory
Adding Xskipper proposal

### CODE REVIEW

**Comment by nakfour** [comment]
This looks great, thanks for submitting @oshritf . I have one comment, is it possible to add links on how to use xskipper? I know you have the api link, but more like a how to get started readme or blog. Thanks

**Comment by oshritf** [comment]
@nakfour done, added direct link to the getting started guide (more for developers on the xskipper project). There's a nice navigation menu on xskipper site. 

We wish to make it easy to ODH JupiterHub Spark notebook - with the focus on working with xskipper in pyspark notebook (direct link in the implementation section)

File: proposal/xskipper.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+## Architecture
+[Xskipper Data Skipping Architecture](https://xskipper.io/master/concepts/extensible/)
+
+## User Use Cases
+Spark big data users who want to boost performance and reduce cost by indexing their data and skipping over irrelevant data
+
+## Implementation
+Xskipper is a Spark extension with [scala and python api](https://xskipper.io/master/api/indexing/). 
+Xskipper jar for the relevant spark version should be added to pyspark packages jars, once the jar is included data can be indexed and dataskipping can be enabled. See [example notebook] (https://github.com/xskipper-io/xskipper/blob/master/notebooks/python/Xskipper%20-%20Python%20Sample.ipynb).
+
```
Comment (nakfour) [inline proposal/xskipper.md:25]: @oshritf please delete space after [example notebook] the link shows as text

File: proposal/xskipper.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+## More info
+
+[IEEE Big Data 2020 paper - Extensible Data Skipping](https://arxiv.org/abs/2009.08150)
+[Adding xskipper support to JupiterHub Spark notebooks](https://github.com/opendatahub-io/odh-manifests/pull/451)
+[Issue](https://issues.redhat.com/projects/ODH/issues/ODH-447)
+[Getting started Guide](https://xskipper.io/master/getting-started/quick-start-guide/)
```
Comment (nakfour) [inline proposal/xskipper.md:31]: @oshritf can we make this a list, its hard to read them all in one line?

File: proposal/xskipper.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+## Architecture
+[Xskipper Data Skipping Architecture](https://xskipper.io/master/concepts/extensible/)
+
+## User Use Cases
+Spark big data users who want to boost performance and reduce cost by indexing their data and skipping over irrelevant data
+
+## Implementation
+Xskipper is a Spark extension with [scala and python api](https://xskipper.io/master/api/indexing/). 
+Xskipper jar for the relevant spark version should be added to pyspark packages jars, once the jar is included data can be indexed and dataskipping can be enabled. See [example notebook] (https://github.com/xskipper-io/xskipper/blob/master/notebooks/python/Xskipper%20-%20Python%20Sample.ipynb).
```
Comment (LaVLaS) [inline proposal/xskipper.md:24]: Please remove the whitespace after `[example notebook]`, it is breaking the link in markdown

**Comment by LaVLaS** [comment]
lgtm

**Comment by nakfour** [comment]
lgtm

**Review by crobby — APPROVED** [review]
seems like a good addition

**Pull request merged**